src/pages/Search/components/GifItem/GifItem.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
memo는 바로 불러와서 사용할 수 있을 것 같아요!
```suggestion
import { memo } from 'react';
```